pycm/curve.py

```diff
@@ -183,6 +188,23 @@ def __repr__(self) -> str:
         """Representation method."""
         return "pycm.ROCCurve(classes: " + str(self.classes) + ")"
 
+    def optimal_thresholds(self) -> Dict[Any, float]:
```

Comment:
We could add a property named `optimal_thresholds` which uses this method and caching to be the same as #577 's description.
If you think it's better to be handled in a separate PR, I can do it in a follow-up PR.